sdks/python/container/Dockerfile

```diff
@@ -70,9 +70,11 @@ RUN ccache --set-config=sloppiness=file_macro && ccache --set-config=hash_dir=fa
 ####
 # Install Apache Beam SDK. Use --no-deps and pip check to verify that all
 # necessary dependencies are specified in base_image_requirements.txt.
+# use ARG pip_extra_options to install release candidates(RCs) of Beam'
```

Comment:
again, you are making an assumption that this file can't reason about. one may supply pip extra options for any purpose.

Reply:
yes. previously the purpose was to add only `--pre` but since we can add any flag to pip installation, i changed the wording.